tasks.py

```diff
@@ -24,6 +24,7 @@ def test(ctx, watch=False, last_failing=False):
     if int(sys.version_info[0]) < 3:
         args.append('--ignore={0}'.format(os.path.join('tests', 'test_py3')))
     retcode = pytest.main(args)
+    benchmark(ctx)
```

Comment:
In the interest of making the tests run quickly, it would be nice to exclude benchmarks by default when running `inv test`. Can you add an argument, e.g. `--benchmark` instead?